src/app/action-card/action-card.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="d-flex justify-content-center align-items-center" style="height: 100vh">
+  <div class="card mx-auto justify-content-md-center" style="width: 50rem">
+    <div class="card-header" style="background-color: white">
+      <h2 [ngStyle]="{color: titleColor}">{{ cardTitle }}</h2>
+      <h3>
+        <button><i class="bi bi-plus-circle"></i></button> Add activity
+      </h3>
+    </div>
+    <ul class="list-group list-group-flush" *ngFor="let feature of features">
+      <li class="list-group-item">
+        <div>
+          <button (click)="toggleMenu()" class="fst-normal">
+            <i class="bi bi-chevron-compact-down fst-normal" *ngIf="showToDoList"></i>
+            <i class="bi bi-chevron-compact-up fst-normal" *ngIf="!showToDoList"></i>
+            {{ feature.featureName }}
+            <span class="badge bg-secondary" *ngIf="!feature.canDelete">Must-have tool</span>
+          </button>
+        </div>
+        <div>
+          <i class="bi bi-pencil-square"></i>
+          <span class="d-inline-block" tabindex="0" data-toggle="popover" data-trigger="hover focus"
+            data-title="Do you want to delete this feature? First create another “must have tool”."
+            data-content="There must be at least one “Must have tool” in your list. If you wish to delete this feature, you first need to create another feature with the same badge."
+            data-container="body" data-bs-custom-class="custom-popover">
+            <button class="btn" type="button" disabled *ngIf="!feature.canDelete">
+              Delete?
+            </button>
+          </span>
+          <i class="bi bi-trash" *ngIf="feature.canDelete"></i>
+        </div>
+      </li>
+      <ul class="list-group todo" *ngIf="feature.toDoList && showToDoList">
```

Comment:
wrap unordered list in ng-container. 
it should be sth like this:
<ng-container *ngIf="feature.toDoList && showToDoList">
<ul>
// code goes here
<ul>
</ng-container>